packages/analytics/metric-cards/package.json

```diff
@@ -0,0 +1,63 @@
+{
+  "name": "@kong-ui-public/metric-cards",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "./dist/metric-cards.umd.js",
+  "module": "./dist/metric-cards.es.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
+    "dist"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/metric-cards.es.js",
+      "require": "./dist/metric-cards.umd.js"
+    },
+    "./package.json": "./package.json",
+    "./dist/*": "./dist/*"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "scripts": {
+    "dev": "cross-env USE_SANDBOX=true vite",
+    "build": "run-s typecheck build:package build:types",
+    "build:package": "vite build",
+    "build:types": "vue-tsc -p './tsconfig.build.json' --emitDeclarationOnly",
+    "build:visualize": "BUILD_VISUALIZER='analytics/metric-cards' vite build -m production",
+    "preview:package": "vite preview --port 4173",
+    "preview": "cross-env USE_SANDBOX=true PREVIEW_SANDBOX=true run-s build:package preview:package",
+    "lint": "eslint '**/*.{js,jsx,ts,tsx,vue}' --ignore-path '../../../.eslintignore'",
+    "lint:fix": "eslint '**/*.{js,jsx,ts,tsx,vue}' --ignore-path '../../../.eslintignore' --fix",
+    "stylelint": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}'",
+    "stylelint:fix": "stylelint --allow-empty-input './src/**/*.{css,scss,sass,less,styl,vue}' --fix",
+    "typecheck": "vue-tsc -p './tsconfig.build.json' --noEmit",
+    "test:component": "BABEL_ENV=cypress cross-env FORCE_COLOR=1 cypress run --component -b chrome --spec './src/**/*.cy.ts' --project '../../../.'",
+    "test:component:open": "BABEL_ENV=cypress cross-env FORCE_COLOR=1 cypress open --component -b chrome --project '../../../.'",
+    "test:unit": "cross-env FORCE_COLOR=1 vitest run",
+    "test:unit:open": "cross-env FORCE_COLOR=1 vitest --ui"
+  },
+  "peerDependencies": {
+    "@kong/kongponents": "^8.22.4",
+    "vue": "^3.2.47"
+  },
+  "devDependencies": {
+    "@kong/kongponents": "^8.22.4",
+    "approximate-number": "^2.1.0",
```

Comment:
@mihai-peteu I believe this needs to be added as a `dependency` and removed as a `devDependency`

Reply:
Done deal! Waiting for the temp package to rebuild, will re-test in a couple of consuming apps.

Reply:
looking good still

<img width="785" alt="Screenshot 2023-02-27 at 4 40 58 PM" src="https://user-images.githubusercontent.com/103061463/221693375-6e3faf9b-8fdb-4a2e-bee7-3607885dc6d4.png">


<img width="707" alt="Screenshot 2023-02-27 at 4 47 44 PM" src="https://user-images.githubusercontent.com/103061463/221693390-fd48b74b-a745-46f9-8a6e-178c0a164f18.png">